talks/_posts/2016-02-05-using-znc.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Never leave IRC again with ZNC
+author: Justin W. Flory
+layout: post
+slides: /talks/2015-fall-using-znc.pdf
+date: 2015-12-04
```

Comment:
You might want to update this date (or remove it, jekyll will use the filename if it isn't there)